components/layout/SiteFooter.tsx

```
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import Link from 'next/link'
+
+const MetaplexLogo: React.FC<React.SVGProps<SVGSVGElement>> = ({ className, ...rest }) => (
+  <svg width="112" height="112" viewBox="0 0 112 112" xmlns="http://www.w3.org/2000/svg" className={className} {...rest}>
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)